src/components/remotecontrol/remotecontrol.js

```diff
@@ -316,25 +353,39 @@ define(['browser', 'datetime', 'backdrop', 'libraryBrowser', 'listView', 'imageL
 
         function updateRepeatModeDisplay(repeatMode) {
             var context = dlg;
-            var toggleRepeatButton = context.querySelector('.repeatToggleButton');
-
-            if ('RepeatAll' == repeatMode) {
-                toggleRepeatButton.innerHTML = "<span class='material-icons repeat'></span>";
-                toggleRepeatButton.classList.add('repeatButton-active');
-            } else if ('RepeatOne' == repeatMode) {
-                toggleRepeatButton.innerHTML = "<span class='material-icons repeat_one'></span>";
-                toggleRepeatButton.classList.add('repeatButton-active');
-            } else {
-                toggleRepeatButton.innerHTML = "<span class='material-icons repeat'></span>";
-                toggleRepeatButton.classList.remove('repeatButton-active');
+            let toggleRepeatButtons = context.querySelectorAll('.repeatToggleButton');
+            const cssClass = 'repeatButton-active';
+            let innHtml = '<span class="material-icons repeat"></span>';
+            let repeatOn = true;
+
+            switch (repeatMode) {
+                case 'RepeatAll':
+                    break;
+                case 'RepeatOne':
+                    innHtml = '<span class="material-icons repeat_one"></span>';
+                    break;
+                case 'RepeatNone':
+                    repeatOn = false;
+                    break;
+                case undefined:
+                    break;
+                default:
+                    throw new TypeError('invalid value for repeatMode');
+            }
+
+            for (const toggleRepeatButton of toggleRepeatButtons) {
+                if (!repeatOn) {
+                    toggleRepeatButton.classList.remove(cssClass);
+                } else {
+                    toggleRepeatButton.classList.add(cssClass);
+                }
```

Comment:
```suggestion
                toggleRepeatButton.classList.toggle(cssClass, repeatOn);
```
In fact, this can be applied in many cases that we have.

Reply:
@dmitrylyzo Didn't know about ``toggle``! This is a life saver! 😱 Thank you!

Will apply it only to the new toggles I added/edited in this PR. Will keep all the ones that were previously in this file intact, don't want to make the diffs any larger. Will take a note of this for the ES6 migration.